.github/workflows/release.yaml

```
@@ -253,6 +253,10 @@ jobs:
             arch: '-A Win32'
           - triplet: x64-windows-static
             arch: '-A x64'
+          - triplet: x86-windows-dynamic
+            arch: '-A Win32'
+          - triplet: x64-windows-dynamic
```

```suggestion
          - triplet: x86-windows
            arch: '-A Win32'
          - triplet: x64-windows
```
Does foo-windows is dynamic by default?

---

https://github.com/libjxl/libjxl/actions/runs/7445929299/job/20255177651?pr=3117